DQM/BeamMonitor/plugins/OnlineBeamMonitor.cc

```diff
@@ -128,6 +128,50 @@ void OnlineBeamMonitor::bookHistograms(DQMStore::IBooker& ibooker,
   bsChoice_->setAxisTitle("Choice", 2);
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+// Handle exceptions for the schema evolution of the BeamSpotOnline CondFormat
+
+// Slightly better error handler
+static void print_error(const std::exception& e) { edm::LogError("BeamSpotOnlineParameters") << e.what() << '\n'; }
+
+// Method to catch exceptions
+template <typename T, class Except, class Func, class Response>
+T try_(Func f, Response r) {
+  try {
+    LogDebug("BeamSpotOnlineParameters") << "I have tried" << std::endl;
+    return f();
+  } catch (Except& e) {
+    LogDebug("BeamSpotOnlineParameters") << "I have caught!" << std::endl;
+    std::cout << "I have caught!" << std::endl;
```

Comment:
No `cout`

Reply:
ah yes I forgot to clean that out after testing locally 😄 I will remove that in the next push

Reply:
solved in [3bb9db9](https://github.com/cms-sw/cmssw/pull/40716/commits/3bb9db9923c8635ab686d222da8785c252d512b4)